### DISCUSSION TITLE
Do we know how long returned links stay valid?

### DISCUSSION THREAD

**Discussion by port19x** [pystardust/ani-cli#1185] [Q&A] [answered]
Title.

**Comment by Derisis13** [comment]
I'd say pretty long, I had one yesterday sitting for a few hours and it played until the very end, even though it wasn't cached completely

**Comment by CoolnsX** [comment, marked as answer]
depends on how providers handle links on their end. some does 8 hours, some whole day and some like 2 hours. It's also depends on CDN cache, like if something is used frequently its link will be longer valid
